### DISCUSSION TITLE
How are you guys getting all the music data?

### DISCUSSION THREAD

**Discussion by Longshoez** [ciderapp/Cider#772] [Q&A] [answered]
I doubt there's an apple music Api so what I'm thinking is that you guys built your app on top of the existing apple music, I'm not calling you out, just curious. the client looks amazing. I've been using apple music as a pwa with that chrome feature that lets you turn pages into standalone apps, the only downside is that it requires you to have chrome open.

**Comment by tva2000hn** [comment, marked as answer]
> I doubt there's an apple music Api

https://developer.apple.com/documentation/musickitjs

**Reply by quacksire** [reply]
eMoToNaIl DaMaGE

**Reply by maikirakiwi** [reply]
Yeah why are you linking this, musickit doesn't exist. There's like no documentation so no one could use it. Btw we should link library to music.apple.com's library because it's so slow right now.
